modules/server/src/config/types.ts

```diff
@@ -143,6 +150,12 @@ export interface TableConfigsInterface {
 	[ConfigProperties.ROW_ID_FIELD_NAME]?: string;
 }
 
+export interface NetworkAggregationInterface {
+	[NetworkAggregationProperties.ENDPOINT]: string;
+	[NetworkAggregationProperties.ARRANGER_FIELD]: string;
+	[NetworkAggregationProperties.NAME]: string;
+}
+
```

Comment:
Might need to expand on the config object to be nested because it's merged into a single object. a key like `name` isn't very unique and there isn't a unique field guard

Reply:
structure is dependant on our json config files. no code enforcement. maybe a future improvement using zod to match a config schema 
eg. if a field is nested, it will be nested in config